tests/Unit/Routes/PlaceRatingRoutesTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+
+namespace tests\Unit\Routes;
+
+use Illuminate\Support\Facades\Route;
+use Tests\TestCase;
+
+class PlaceRatingRoutesTest extends TestCase
```

Comment:
Firstly you don't need to test existing routes, because if you make in tests above request on non existent route a test won't be passed. And second you cannot mention address your environment in the public code `http://localhost` because it won't be executed on stage and production servers, or in the environment your teammates

Reply:
i implemented this unit test on early stage of feature development, when i cannot set-up my feature test. i would rather prefer to replace `http://localhost` with `URL::to('/')` in this test?